tests/test_market_quote.py

```
@@ -0,0 +1,159 @@
+    assert ev.inventory == Inventory()
+    assert len(ev.intents) == 2
+    assert ev.why == ""
+
+
+def test_cli_adapter_passes_require_rewards_off_like_decide():
+    """decide() fetches with require_rewards=False; the step must receive the
+    market it would get, not re-fetch."""
+    m = _Market()
+    seen = {}
+
+    def fetch_market(cid):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `fetch_market`

(ANN202)